.github/workflows/node.js.yml

```diff
@@ -10,20 +10,80 @@ on:
     branches: [ master ]
 
 jobs:
+# building job
   build:
-
     runs-on: ubuntu-latest
+    name: Build
 
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x, 15.x]
 
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Building on Node.js ${{ matrix.node-version }} 🛠️
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - run: npm ci # equivalent of npm ci
+    - run: npm build
+      env:
+        CI: true
+# testing job
+  test:
+    runs-on: ubuntu-latest
+    name: Test
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Testing on node ${{ matrix.node-version }} 🧪
+      uses: actions/setup-node@v1
+      with:
+          node-version: ${{ matrix.node-version }}
     - run: npm ci
-    - run: npm run build --if-present
     - run: npm test
+# # coverage uploading job
```

Comment:
@z3bi @sgtsquiggs I also suggest to move code coverage to GitHub actions and eliminate the need for travis.

I have set up the job already if you guys want that just uncomment and add `CODECOV_TOKEN ` as well as adjust the directory (forgot to do that)